sssom/writers.py

```diff
@@ -125,7 +115,8 @@ def write_owl(
         serialisation = SSSOM_DEFAULT_RDF_SERIALISATION
 
     graph = to_owl_graph(msdf)
-    graph.serialize(destination=filename, format=serialisation)
+    t = graph.serialize(format=serialisation, encoding="utf-8")
```

Comment:
this is annoying - rdflib only writes to bytes (though I heard they got rid of this in rdflib 6 #138!!!)